ansible_events/drools/durable_rules_engine.py

```diff
@@ -113,18 +15,35 @@ def __init__(self, *args, **kwargs):  # real signature unknown
     """list of weak references to the object (if defined)"""
 
 
+def make_instance():
+    jpy_classpath = os.environ.get("ANSIBLE_EVENTS_DROOLS_JPY_CLASSPATH")
+    if jpy_classpath:
+        from ansible_events.drools.jpy_durable_rules_engine import (
+            JpyDurableRulesEngine,
+        )
+
+        return JpyDurableRulesEngine(jpy_classpath)
+
+    from ansible_events.drools.rest_durable_rules_engine import (
+        RestDurableRulesEngine,
+    )
+
+    return RestDurableRulesEngine(
+        os.environ.get("ANSIBLE_EVENTS_DROOLS_HOST", "http://localhost:8080")
+    )
+
+
 # exported methods
-_instance = DurableRulesEngine(
-    os.environ.get("ANSIBLE_EVENTS_DROOLS_HOST", "http://localhost:8080")
-)
+
+_instance = make_instance()
 
 
 def abandon_action(*args, **kwargs):  # real signature unknown
     pass
 
 
 def assert_event(session_id, serialized_fact):
-    return _instance.assert_fact(session_id, serialized_fact)
+    return _instance.assert_event(session_id, serialized_fact)
```

Comment:
@evacchi This is causing test failures when multi event is involved.

```
RULES_ENGINE=drools pytest tests/test_examples.py::test_15_multiple_events_all
```
Since the facts keep the events around but I think with events they are getting removed.

Reply:
yes, saw the comments in #134, obviously same underlying issue